### ISSUE TITLE
Big Sur Crashing with EFI v1.9.0

### ISSUE THREAD

**Issue by mmenendezg** [profzei/Matebook-X-Pro-2018#137]
Hello people, 

I have had recently a lot of issues when using the MBP playing some video or watching video tutorials and programming at the same time. I thought It may be due to the use of iGPU of Intel's CPU instead of the dGPU, and the iGPU capabilities, but this has happen under several conditions, even when I was not stressing the CPU at all. 

I do not remember having this inconveniences in the past or the previous releases. I am going to try to downgrade to 1.8.0 and if I find any relevant discover I will update.

**Comment by profzei** [comment]
@mmenendezg Ok!
What I can say is that my system is very rock solid with the amount of workload you described...

**Comment by mmenendezg** [comment]
Hello @profzei , 

I was not able to solve the issue by changing of EFI version. I decided to make a fresh installation of Big Sur (I lost my anime though) and I am still testing, and no further crashing by now. 

My guess is that it was related to some configurations in my development environment, but cannot give more details since I did not verify deeply, therefore I am lacking the details. 

If the crashing comes back, will let you know. Everything stable for now.
